CUDADataFormats/TrackingRecHit/interface/TrackingRecHit2DHeterogeneous.h

```diff
@@ -43,7 +44,7 @@ class TrackingRecHit2DHeterogeneous {
 
 private:
   static constexpr uint32_t n16 = 4;
-  static constexpr uint32_t n32 = 9;
+  static constexpr uint32_t n32 = 10;
```

Comment:
As suggested by @slava77 [here](https://github.com/cms-sw/cmssw/pull/31721/files#r521072888), could you add a comment about what `4` and ~`9`~ `10` stand for ?

Reply:
should be pretty obvious from the code where it is used.
They are of course the number of element with size 16 and size 32 respectively.
I will add a comment.